Automattic/jetpack
# Sharing: fix icon font display

This is a follow-up of #31873.

## Proposed changes:

The icon font's base64-encoded string wasn't correct the last time around. this should fix it.

### Other information:


## Jetpack product discussion

* p1689640150536069-slack-CBG1CP4EN
* https://github.com/Automattic/jetpack/pull/31873#issuecomment-1639111181

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings > Sharing
* Enable sharing buttons
* Go to Settings > Sharing, and enable a few buttons
* Check that they look good on the frontend.
* Check that on wpcom simple too.
